src/HyperFormula.ts

```diff
@@ -3896,7 +3952,8 @@ export class HyperFormula implements TypedEmitter {
   }
 
   /**
-   * Returns a normalized formula.
+   * Parses and then unparses a formula.
+   * Returns a normalized formula (e.g. normalizes the capitalization of sheet names, function names and cell addresses).
```

Comment:
and named expressions, I think 🤔 "normalizes" means it will use the original capitalization they were registered with

Reply:
@wojciechczerniak updated: 88e37ec